gas/config/tc-aarch64.c

```diff
@@ -10250,9 +10250,10 @@ struct aarch64_option_abi_value_table
   enum aarch64_abi_type value;
 };
 
-#ifdef OBJ_ELF
 static const struct aarch64_option_abi_value_table aarch64_abis[] = {
+#ifdef OBJ_ELF
   {"ilp32",		AARCH64_ABI_ILP32},
+#endif
   {"lp64",		AARCH64_ABI_LP64},
```

Comment:
We know the only valid data model for aarch64 coff/pe is llp64.
What is the use-case where "-mabi=lp64" would be valid for coff? 
If we want to allow tests to explicitly set a memory model shouldn't "-mabi=llp64"" exist? (even if it is the default.)
The original patch author believed the memory model should not be optional for coff/pe as llp64 is the only valid option.

Reply:
looks good to me  (y)